build_projects/dotnet-host-build/project.json

```diff
@@ -5,13 +5,13 @@
     "emitEntryPoint": true
   },
   "dependencies": {
-    "NETStandard.Library": "1.6.0-rc3-24201-00",
-    "Microsoft.NETCore.Runtime": "1.0.2-rc3-24201-00",
-    "Microsoft.CSharp": "4.0.1-rc3-24201-00",
-    "System.Dynamic.Runtime": "4.0.11-rc3-24201-00",
-    "System.Reflection.Metadata": "1.3.0-rc3-24201-00",
-    "System.Runtime.Serialization.Primitives": "4.1.1-rc3-24201-00",
-    "System.Xml.XmlSerializer": "4.0.11-rc3-24201-00",
+    "NETStandard.Library": "1.6.0-rc3-24203-00",
+    "Microsoft.NETCore.Runtime": "1.0.2-rc3-24203-00",
```

Comment:
@eerhardt That will happen automatically here because it uses the full list of package-versions, right? From the naming of the repo it looks like core-setup already draws from multiple lists, CoreFX and Roslyn... is that right? (Then it'd just need to pull from the CoreCLR list too.)